stock_inventory_valuation_location/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+hbto@vauxoo.com
```

Comment:
It should be:

Full name <email>

Also I don't see your own name here :smile:

Reply:
Change applied. Thanks